src/data/set/pairwise.lean

```diff
@@ -421,3 +423,30 @@ end set
 
 lemma pairwise_disjoint_fiber (f : ι → α) : pairwise (disjoint on (λ a : α, f ⁻¹' {a})) :=
 set.pairwise_univ.1 $ set.pairwise_disjoint_fiber f univ
+
```

Comment:
```suggestion
```

Reply:
Oh you could have committed this yourself. A bit late now 😅